dynamo/tools/dimension_reduction.py

```diff
@@ -45,7 +45,7 @@ def reduceDimension(
         dims:
             The list of dimensions that will be selected for clustering. If `None`, all dimensions will be used.
         n_pca_components:
-            Number of PCA components.
+            run dimension reduction on `n_pca_components` PC transformed data. It triggers generating n_pca_components if n_pca_components is larger than the existing #PC in adata.
```

Comment:
Number of PCA components that will be used to perform the non-linear dimension reduction. If `n_pca_components` is larger than the existing #PC in `adata.obsm['X_pca']`, pca will be rerun with `n_pca_components` PCs requested.

Reply:
fixed and please see the updated docs